multi-agent/lib/mas/resources/builtin_aware_service.py

```
@@ -0,0 +1,532 @@
+    ) -> ElementValidationResult:
+        try:
+            return self.validate_resource(
+                rid=rid, caller=caller, user_id=user_id,
+                timeout_seconds=timeout_seconds,
+                credential_user_id=credential_user_id,
+            )
+        except KeyError:
+            return ElementValidationResult.create_error(
+                rid=rid, error=f"Resource not found: {rid}",
+            )
+        except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)